examples/forms/select_with_value_in_schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "body": {
+    "form": {
+      "расчетныйсчет": {
+        "block": "selectWithValue",
+        "name": "расчетныйсчет",
+        "label": "Расчетный счет",
+        "labelWidth": 3,
+        "values": []
+      }
+    },
+    "schema": [
+      "расчетныйсчет"
+      ]
+  }
+}
```

Comment:
Text files should end with newline symbols

[Further reading](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)